hilti/runtime/include/types/bytes.h

```diff
@@ -315,9 +319,11 @@ class Bytes : protected std::string {
      * specified character set.
      *
      * @param cs character set to assume the binary data to be encoded in
+     * @param errors how to handle errors when decoding the data
      * @return UTF8 string
      */
-    std::string decode(bytes::Charset cs) const;
+    std::string decode(bytes::Charset cs,
```

Comment:
With replacement here users could decode e.g., to `ASCII` and end up with a non-ASCII result (containing UTF8 substitute characters). I think that if we want to keep this API we really need to specify the replacement character very explicitly as part of the API so users can sanitize results after the fact.

Reply:
Was incorrect on this one.